server/controllers/api_controller.go

```diff
@@ -40,6 +40,7 @@ type APIRequest struct {
 	Projects   []string
 	Paths      []struct {
 		Directory string
+		Flags     []string
```

Comment:
I think this should be called `ExtraArgs` https://www.runatlantis.io/docs/custom-workflows.html#custom-run-command to match the docs and naming.

Reply:
Gotcha, thats a fair point.  Originally choose `Flags` since it matches the `Flags` attribute in the `CommentCommand` struct.
https://github.com/runatlantis/atlantis/blob/main/server/events/event_parser.go#L104 

Do you still feel it would be more clear to call this `ExtraArgs`?  I'm happy to make that change.

Reply:
Yes please  i agree with @jamengual. Lets go with consistency here